File: 0217-contains-duplicate/0217-contains-duplicate.swift
class Solution {
    func containsDuplicate(_ nums: [Int]) -> Bool {
        var dic = [Int:Int]()
        for i in nums {
            if dic[i] == nil {
                dic[i] = 1
            } else {
                return true
            }
        }
        return false
    }
}

File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:

        return len(nums) != len(set(nums))
    
        # or
        # return True if len(set(nums)) < len(nums) else False
        # or
        
        # d={}
        # for i in nums:
        #     if i not in d:
        #         d[i]=1
        #     else:
        #         return True
        # return False
        
        
# Just need to return true if any value is more than once.
# We can also just compare normal len and set len.